#Header header    # ROS Message headaer

uint8 target      # Target

float32 vbus      # Bus voltage

int32 error0      # Axis 0 error
int32 state0      # Axis 0 state
float32 vel0      # Axis 0 velocity 
float32 pos0      # Axis 0 encoder position
float32 curr0B    # Axis 0 motor channel B current
float32 curr0C    # Axis 0 motor channel C current
float32 temp0     # Axis 0 invertr temperature

int32 error1      # Axis 1 error
int32 state1      # Axis 1 state
float32 vel1      # Axis 1 velocity 
float32 pos1      # Axis 1 encoder position
float32 curr1B    # Axis 1 motor channel B current
float32 curr1C    # Axis 1 motor channel C current
float32 temp1     # Axis 1 inverter temperature

